src/main/java/seedu/address/logic/commands/SellCommand.java

```diff
@@ -0,0 +1,116 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_GOOD_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_QUANTITY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TRANSACTION_DATE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.good.Good;
+import seedu.address.model.good.GoodName;
+import seedu.address.model.good.GoodQuantity;
+
+/**
+ * Sells the stated quantity of the good from the given supplier on the provided transaction date.
+ * User will not be allowed to sell more than their current stock in inventory.
+ */
+public class SellCommand extends Command {
+    public static final String COMMAND_WORD = "sell";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sells the stated "
+            + "quantity of the good on the provided transaction date. "
+            + "Users are not allowed to sell more than they have in stock. "
+            + "Parameters: "
+            + PREFIX_GOOD_NAME + "GOOD_NAME "
+            + PREFIX_QUANTITY + "QUANTITY "
+            + PREFIX_TRANSACTION_DATE + "TRANSACTION_DATE "
```

Comment:
I think you are missing a newline after TRANSACTION_DATE

Reply:
Added. Also added it to BuyCommand since newline was missing there too.